aws/your-cluster.tfvars.template

```diff
@@ -10,13 +10,13 @@ allowed_roles = ["<DEPLOY-ROLE>"]
 # # ============================================================================================================
 
 # # Configured for unmanaged private subnets created by IT
-vpc_cidr = "<VPC-CIDR>"   #  "10.144.0.0/12"   # managed or unmanaged for EKS and EFS.  single block only
 
 vpc_name = "<VPC-NAME>"   # "DEV-WF-SC-SB"
 
 public_subnet_names = ["<PUBLIC-SUBNET-NAMES-OR-PATTERNS>"]  # ["DEV-WF-SC-SB-Public-*"]
 
 private_subnet_names = ["<PRIVATE-SUBNET-NAMES-OR-PATTERNS>"]  # ["DEV-WF-SC-SB-DMZ-*"]
 
-cluster_endpoint_public_access_extra_cidrs = []  # Public EKS API endpoint only reachable by NATs and these.
+cluster_sg_name = ""  # "jmiller-cluster-sg" modeled after Additional worker sg
```

Comment:
want to remove personal info from the comment here?

Reply:
We could but for now I'm documenting the as-built sg's because we need to pass those onto ITSD and there's no up-to-date  terraform for them.

Reply:
I changed "jmiller" to "user" everywhere in your-cluster...